chef_master/source/chef_license_accept.rst

```diff
@@ -157,119 +185,107 @@ Specify the following argument:
+    }
 
-Chef Terraform Provisioner
+Terraform Chef Provisioner
 -----------------------------------------------------
-The license can be accepted via the Chef Infra config file, which is specified by the ``client_options`` Terraform
-config:
+The license can be accepted via the Chef Infra Client config file, which is specified by the ``client_options`` `Terraform provisioner config <https://www.terraform.io/docs/provisioners/chef.html#client_options-array->`__:
 
 .. code-block:: none
 
     provisioner "chef" {
-      ...
       client_options = ["chef_license 'accept'"]
-      # If left out this will default to latest
-      version         = "15"
     }
 
 Pin to Chef 14
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-In your Terraform provisioner config:
+In your `Terraform provisioner config <https://www.terraform.io/docs/provisioners/chef.html#version-string->`__, include:
 
 .. code-block:: none
 
     provisioner "chef" {
-      ...
-      version         = "14.12.3"
+      version = "14.12.3"
     }
 
-
-Packer
+Terraform Habitat Provisioner
 -----------------------------------------------------
-Use a custom `Chef configuration template <https://www.packer.io/docs/provisioners/chef-client.html#chef-configuration>`__.
-In your provisioners config:
+Default behavior of this provisioner is to install the latest version of Habitat. `Documentation for this provisioner <https://www.terraform.io/docs/provisioners/habitat.html>`__ will be updated in the near future once the provisioner is updated with options to accept license. For the time being, the provisioner can be pinned to a prior Habitat version as below.
 
-.. code-block:: json
+Pin to Chef Habitat 0.79
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In your `Terraform provisioner config <https://www.terraform.io/docs/provisioners/habitat.html#version-string->`__, include:
 
-    {
-      "type":              "chef-client",
-      "config_template":   "path/to/client.rb"
+.. code-block:: none
+
+    provisioner "habitat" {
+      version = "0.79.1"
     }
 
-In ``path/to/client.rb``:
+Vagrant
+-----------------------------------------------------
+This license acceptance can be done via the arguments API:
 
 .. code-block:: ruby
 
-    ...
-    chef_license "accept"
+   config.vm.provision "chef_zero" do |chef|
+     chef.arguments = "--chef-license accept"
+   end
 
-You may also add it to the `execute_command <https://www.packer.io/docs/provisioners/chef-client.html#execute_command>`__, but this is not backwards-compatible, so it is not suggested.
+See `<https://www.vagrantup.com/docs/provisioning/chef_common.html#arguments>`__ for details.
+This argument is not backwards-compatible so instead users can use the `custom config path <https://www.vagrantup.com/docs/provisioning/chef_common.html#custom_config_path>`__ and point at a local file, which specifies the ``chef_license`` config.
```

Comment:
"this argument" is a bit vague in this sentence. Are you talking about the `chef.arguments`? Would "this argument" be considered an API call? Also, at what point is "this argument" no longer backwards-compatible?

Reply:
@mjingle Latest commit hopefully clarified that sentence - LMK if it does not